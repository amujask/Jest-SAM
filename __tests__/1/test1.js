import request from 'supertest'


const baseUrl = 'https://jsonplaceholder.typicode.com/'

describe('todos endpoint', () => {
    it('should return 200', async () => {
        const response = await request(baseUrl).get('todos/1')
        expect(response.statusCode).toBe(200)
        console.log(response.body)
        expect(response.body).toMatchSnapshot(
            {
                userId: expect.any(Number),
                id: expect.any(Number),
                completed: expect.any(Boolean)
            }
        )
    })
    
})
