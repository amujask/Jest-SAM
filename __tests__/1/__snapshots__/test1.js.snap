// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`todos endpoint should return 200 1`] = `
{
  "completed": Any<Boolean>,
  "id": Any<Number>,
  "title": "delectus aut autem",
  "userId": Any<Number>,
}
`;
